From: Dallas Strouse <dastrouses@gmail.com>
Date: Fri, 11 Aug 2023 15:20:53 +0000
Subject: util: Export trySpawn()

It was private before, and so any users of it (including the
Background Apps menu) would fail to utilize it, making apps fail
to get told to shut down from the Background Apps menu.

https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6893

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2885>
(cherry picked from commit 7e18b4d0b20eab6bef677832b96d33bcc540e19a)
---
 js/misc/util.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/misc/util.js b/js/misc/util.js
index 971c630..0201fd1 100644
--- a/js/misc/util.js
+++ b/js/misc/util.js
@@ -117,7 +117,7 @@ export function spawnApp(argv) {
  * Runs argv in the background. If launching argv fails,
  * this will throw an error.
  */
-function trySpawn(argv) {
+export function trySpawn(argv) {
     let success_, pid;
     try {
         [success_, pid] = GLib.spawn_async(
